mobile/src/feeds/home-feeds-screen.tsx

```diff
@@ -69,22 +72,24 @@ class HomeFeedsScreenImpl extends React.Component<FeedsScreenProps & FeedScreenD
     }
 
     public render() {
-
-        const feeds = this.sortedFeeds();
+        const feeds = this.sortedFeeds().filter(this.feedPredicate);
```

Comment:
won't get here because if you change filter only, shouldComponentUpdate still returns false.